saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/Dictionaries.scala

```diff
@@ -0,0 +1,19 @@
+package edu.illinois.cs.cogcomp.saulexamples.nlp.SpatialRoleLabeling
+
+import scala.collection.immutable.HashSet
+
+/** Created by taher on 8/4/16.
+  */
+object Dictionaries {
+
+  val prepositions = HashSet(
+    "about", "above", "across", "after", "against", "along", "among", "around", "as", "at", "before",
+    "behind", "beneath", "beside", "between", "by", "down", "during", "for", "from", "in", "inside",
+    "into", "like", "of", "off", "on", "onto", "over", "round", "through", "to", "towards", "with"
+  )
+
+  def isPreposition(word: String): Boolean = {
+    val w = if (word == null) "" else word.toLowerCase.trim
```

Comment:
Unlike Java, in Scala it is always recommended to use `Option`s. 
Here you can have:
```scala 
def isPreposition(wordOpt: Option[String]): Boolean = {
   wordOpt match { 
      case Some(word) => prepositions.contains(word.toLowerCase.tim)
      case None => false 
   }   
} 
```